testapps/ApplicationWithParseErrors/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1 @@
+﻿<a href="@DateTime.()>Hello world</a>
```

Comment:
I don't think this is an error. The only two errors are from `Index.cshtml` and `_ViewImports.cshtml`